docs/source/support/releases.rst

```diff
@@ -0,0 +1,86 @@
+.. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Releases and Versioning
+=======================
+
+.. _versioning:
+
+Versioning
+----------
+
+All DAML components follow `Semantic Versioning <https://semver.org/>`_. In short, this means that there is a well defined "public API" changes or breakages to which are indicated by the version number.
+
+Given a version number MAJOR.MINOR.PATCH, increment the:
```

Comment:
Does the increment happen on each change? Or only when the next snapshot is released?

Reply:
This was a copy&paste job from https://semver.org/. The way they had phrased it was talking to the developer. I've now rephrased it to talk to the consumer. Do you think it addresses your comments?

Reply:
Yes, thank you.